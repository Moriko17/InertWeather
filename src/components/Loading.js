import React from "react";
import { BeatLoader } from "react-spinners";

export default function() {
  return (
    <div className="loading">
      <BeatLoader />
    </div>
  );
}
