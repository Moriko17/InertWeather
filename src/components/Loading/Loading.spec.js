import React from "react";
import renderer from "react-test-renderer";
import Loading from "./Loading";

describe("Loading renders right", () => {
  it("Renders correctly", () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
