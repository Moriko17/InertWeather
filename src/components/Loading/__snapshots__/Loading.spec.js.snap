// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders right Renders correctly 1`] = `
<div
  className="loading"
>
  <div
    className="css-0"
  >
    <div
      className="css-110do72"
    />
    <div
      className="css-1by9rry"
    />
    <div
      className="css-110do72"
    />
  </div>
</div>
`;
